laboratoriosIndex.html

```diff
@@ -0,0 +1,377 @@
+
+      <div class="collapse navbar-collapse w-100" id="sidebar-content">
+        <div class="nav-container d-flex flex-column w-100 h-100 justify-content-start justify-content-sm-between">
+          <ul class="nav flex-column">
+            <li class="nav-item">
+              <a class="nav-link" href="/">
+                <i class="fa-solid fa-table-cells-large fa-lg"></i>
+                Dashboard
+              </a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="/solicitudes.html">
+                <i class="fa-solid fa-inbox fa-lg"></i>
+                Solicitudes
+              </a>
+            </li>
+            <hr />
+            <li class="nav-item">
+              <a class="nav-link active" href="/laboratoriosIndex.html">
+                <i class="fa-solid fa-desktop fa-lg"></i>
+                Laboratorios
+              </a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="/usuarios">
+                <i class="fa-solid fa-user fa-lg"></i>
+                Usuarios
+              </a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="/informes">
+                <i class="fa-solid fa-file-lines fa-lg"></i>
+                Informes
+              </a>
+            </li>
+          </ul>
+          <ul class="nav flex-column">
+            <li class="nav-item">
+              <a class="nav-link" href="/ajustes">
+                <i class="fa-solid fa-gear fa-lg"></i>
+                Ajustes
+              </a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link account-link" href="/cuenta">
+                <img src="/assets/chad.png" class="account-img" alt="Chad" />
+                <div class="account-details">
+                  <span class="account-name">Yamil Guzmán</span>
+                  <span class="account-role">Coordinador</span>
+                </div>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <main class="container p-4">
+      <div class="nav-top">
+        <div class="nav-item">
+          <h3 class="lab" data-filter="lab">Laboratorios</h3>
+        </div>
+        <div class="nav-item">
+          <button type="button" class="add" data-toggle="modal" data-target="#myModal">
+            <i class="fa-solid fa-circle-plus"></i> Añadir Laboratorio
+          </button>
+          <div class="input-group">
+            <span class="input-group-prepend">
+              <div class="input-group-text bg-transparent border-right-0"><i class="fa fa-search"></i></div>
+            </span>
+            <input id="myInput" class="form-control py-2 border-left-0 shadow-none" type="search"
+              placeholder="Buscar...">
+          </div>
+        </div>
+      </div>
+
+      <a href="laboratorio.html">
+        <div class="cards-container">
+          <div class="card">
```

Comment:
En el caso de los Cards, debes de usar los ejemplos propuestos en el `index.html`.

```suggestion
          <div class="card border-0 rounded-xl shadow p-3 d-flex align-items-center">
```

Reply:
Same, me falla entre los breakpoints de medium (768px) y extra large (1200px)